handlers/apps.go

```diff
@@ -47,88 +54,110 @@ func (p *appParams) extract(r *http.Request) {
 }
 
 // ClustersApps is the API handler to fetch all the apps to be displayed, related to a single cluster
-func ClustersApps(w http.ResponseWriter, r *http.Request) {
-	query := r.URL.Query()
-	namespacesQueryParam := query.Get("namespaces") // csl of namespaces
-	p := appParams{}
-	p.extract(r)
-
-	businessLayer, err := getBusiness(r)
-	if err != nil {
-		RespondWithError(w, http.StatusInternalServerError, "Apps initialization error: "+err.Error())
-		return
-	}
+func ClustersApps(
```

Comment:
Would it make sense to have a naming convention that makes it easy to recognize the handlers `ClusterAppsHandler`?  Maybe the package is sufficient.  If nothing else, let's take this opportunity to remove the double-plural on this one.

Reply:
Sure I can change that too

Reply:
Did you push changes?